src/plugins/workspace/server/saved_objects/integration_tests/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -51,6 +51,8 @@ const repositoryKit = (() => {
 
 const permittedRequest = httpServerMock.createOpenSearchDashboardsRequest();
 const notPermittedRequest = httpServerMock.createOpenSearchDashboardsRequest();
+const groupIsDashboardAdminRequest = httpServerMock.createOpenSearchDashboardsRequest();
+const UserIdIsDashboardAdminRequest = httpServerMock.createOpenSearchDashboardsRequest();
```

Comment:
Prefer renaming to `userIdIsDashboardAdminRequest `

Reply:
There is no relevant logic of users/groups in the wrapper, so I uniformly use the `dashboardAdminrequest` name in the test file.